include/NaluCommBufferV.hpp

```diff
@@ -67,6 +67,7 @@ class CommBufferV {
     }
 
     unsigned char* raw_buffer() { return data_buffer.data(); }
```

Comment:
Any particular reason why this is `unsigned char`?  `char` seems to work fine here.

Also, why not just call this method `data()`?